application/main.py

```diff
@@ -794,6 +794,31 @@ def get_model(fn):
         return response
     else:
         return send_file(path)
+    
+@application.route('/preprocess_bsdd/<id>', methods=['GET'])
```

Comment:
I propose sth like this:

```suggestion
@application.route('/api/bsdd/statistics/<id>', methods=['GET'])
```